src/http/api/v1/exports.ts

```diff
@@ -90,7 +91,8 @@ export const createHandler = (config: HttpApiConfig): express.Router => {
       applications,
       votes,
       config.passportProvider,
-      {}
+      {},
+      defaultProportionalMatchOptions
```

Comment:
Here we're grabbing something from the calculator module (`import { defaultProportionalMatchOptions } from '../../../calculator/options.js' `) and passing it back to the calculator, unmodified. Since it's a default, the calculator shouldn't need to be told about it, so I'd opt for leaving it `undefined`, letting `getVotesWithCoefficient` fill in the blank, and dropping a module dependency.

That said, `getVotesWithCoefficient` could really use a move from positional arguments to a data object here.

Reply:
@bard good ideas, I removed positional args and set the proportionalMatchingOptions optional using the default by..default